src/Model/GServer.php

```diff
@@ -339,7 +339,7 @@ public static function detect(string $url, string $network = '')
 	 * @param string $server_url address of the server
 	 * @throws \Friendica\Network\HTTPException\InternalServerErrorException
 	 */
-	private static function discoverRelay(string $server_url)
+	public static function discoverRelay(string $server_url)
```

Comment:
Is this change intended or was is just for testing?

Reply:
Good eye.